### PR TITLE
Set up code coverage

### PR DESCRIPTION
**Pull request by vlasiak**
An opportunity to generate, view and share code coverage report.

### CODE REVIEW

**Comment by vlasiak** [comment]
@sergeyradkevich, please look at the [code coverage report](https://stormy-mesa-7604.herokuapp.com).
Credentials for authorisation: `login`: `reviewer`, `password`: `secret`.

**Comment by sergeyradkevich** [comment]
@vlasiak, Can you please customize the report so it displays coverage grouped by the type of source code like in [this example](https://camo.githubusercontent.com/5d9b874a778392ff6180d297faa362f0b32fcd62/687474703a2f2f636f6c737a6f776b612e6769746875622e636f6d2f73696d706c65636f762f6465766973655f726573756c742d302e352e332e706e67) (see tabs).

**Comment by vlasiak** [comment]
@sergeyradkevich, updated. [Link](https://stormy-mesa-7604.herokuapp.com).

File: test/test_helper.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
+require 'simplecov'
+SimpleCov.start do
+  add_group 'Controllers', 'app/controllers'
+  add_group 'Helpers', 'app/helpers'
+  add_group 'Mailers', 'app/mailers'
+  add_group 'Lib', 'lib'
+  add_group 'Models', 'app/models'
+end
```
Comment (sergeyradkevich) [inline test/test_helper.rb]: @vlasiak, OK. What about benefits of usage of [predefined](https://github.com/colszowka/simplecov#profiles) `rails` profile?  Doesn't it do the same configuration (and even more)? What do you think?

**Comment by vlasiak** [comment]
@sergeyradkevich, updated. [Link](https://stormy-mesa-7604.herokuapp.com).

**Pull request merged**